src/Language/PureScript/Docs/AsMarkdown.hs

```diff
@@ -67,6 +67,9 @@ codeToString = outputWith elemAsMarkdown
   elemAsMarkdown Space          = " "
   elemAsMarkdown (Symbol _ x _) = x
 
+  -- roles aren't rendered in markdown
+  elemAsMarkdown (Role _) = ""
```

Comment:
Just want to confirm that this is ok. I think most people don't look at the markdown docs anymore.

Reply:
Added in latest commit.

Reply:
No. Originally, I did, but then see [this comment](https://github.com/purescript/purescript/pull/4121#issuecomment-885409460) for why we didn't move forward with that idea. The last few commits in this PR reverted the commits that added this.

Reply:
Ah, thanks!